### PR TITLE
Fix requires to make gem work

### PR DESCRIPTION
**Pull request by bfcoder**
Remove require_all gem
Require each model individually
Remove byebug requires

This fixes the gem to work.

### CODE REVIEW

File: lib/senkyoshi/xml_parser.rb [hunk 1]
```diff
@@ -1,6 +1,44 @@
-require "require_all"
-require_all "lib/senkyoshi/models/**/*.rb"
```
Comment (nick-benoit14) [inline lib/senkyoshi/xml_parser.rb:2]: What is the motivation for moving away from require_all?

Comment (bfcoder) [reply lib/senkyoshi/xml_parser.rb:2]: because it doesn't work with bundled gems. :]

Comment (bfcoder) [reply lib/senkyoshi/xml_parser.rb:2]: It's also better practice to require each file you need in a gem. Rather than doing a wildcard.

Comment (nick-benoit14) [reply lib/senkyoshi/xml_parser.rb:2]: Okie doke. Just curious. That sounds good though!

**Review by nick-benoit14 — APPROVED** [review]
Looks good to me. Nicely done!

**Pull request merged**